wazo_calld/plugins/transfers/stasis.py

```diff
@@ -282,13 +282,11 @@ def clean_bridge(self, channel, event):
         logger.debug('cleaning bridge %s', bridge.id)
         try:
             self.ari.channels.get(channelId=channel.id)
```

Comment:
channels.get can be removed since we are not doing anything with the result anymore

Reply:
@sduthil Do you think that this comment should be addressed now?

Reply:
Yes, thank you for the ping. It's fixed now.